src/sms_inbound.js

```diff
@@ -149,12 +149,12 @@ go.app = function() {
                                     return self.states.create("state_opt_in_enter");
                                 case "BABY": case "USANA": case "SANA": case "BABA":
                                 case "BABBY": case "LESEA": case "BBY": case "BABYA":
-                                case "OBABY": case "NGWANA": case "BABY BOY":
-                                case "BABY GIRL":
+                                case "OBABY": case "NGWANA": case "BABYBOY":
+                                case "BABYGIRL":
```

Comment:
I noticed that these keywords have spaces in them, but in our processing of the inbound message, we remove all non-word characters, which include spaces. So we should be comparing the message to keywords without spaces.